src/continuation/continuation_featurizing_find_and_match.jl

```diff
@@ -0,0 +1,148 @@
+- `info_extraction = identity`: A function that takes as an input an attractor (`StateSpaceSet`)
+  and outputs whatever information should be stored. It is used to return the
+  `attractors_info` in [`continuation`](@ref). Note that the same attractors that
+  are stored in `attractors_info` are also used to perform the matching in
+  [`match_continuation!`](@ref), hence this keyword should be altered with care.
+- `seeds_from_attractor`: A function that takes as an input an attractor and the number of
+  initial conditions to sample from the attractor, and returns a vector with the sampled initial
+  conditions. By default, we just select the initial conditions randomly from the attractor.
+
+## Description
+
+At the first parameter slice of the continuation process, attractors and their fractions
+are found as described in the [`AttractorsViaFeaturizing`](@ref) mapper identifying groups
+of features. At each subsequent parameter slice, initial conditions are seeded from the
+ previously found attractors. They are put together with the pre-generated `ics` given to
+ `continuation`. Then, all of them are passed onto [`basins_fractions`](@ref) for
+ [`AttractorsViaFeaturizing`](@ref), which featurizes and groups them to identify the
+ attractors. 
+
+Naturally, during this step new attractors may be found. Once the basins fractions are
+computed, the parameter is incremented again and we perform the steps as before.
+
+This process continues for all parameter values. After all parameters are exhausted,
+the found attractors (and their fractions) are "matched" to the previous ones.
+I.e., their _IDs are changed_, so that attractors that are "similar" to those at a
+previous parameter get assigned the same ID.
+Matching is done by the [`match_continuation!`](@ref) function and is an _orthogonal_
+step. This means, that if you don't like the initial
+outcome of the matching process, you may call [`match_continuation!`](@ref) again
+on the outcome with different matching-related keywords.
+You do not need to compute attractors and basins again!
+
+Matching is a very sophisticated process that can be understood in detail by reading
+the docstrings of [`match_statespacesets!`](@ref) first and then [`match_continuation!`](@ref).
+Here is a short summary: attractors from previous and current parameter are matched
+based on their "distance". By default this is distance in state space, but any measure of
+"distance" may be used, such as the distance between Lyapunov spectra.
+Matching prioritizes new->old pairs with smallest distance: once these are matched
+the next available new->old pair with smallest distance is matched, until all new/old
+attractors have been matched. The `threshold` keyword establishes that attractors with
+distance > `threshold` do not get matched.
+Additionally, use `use_vanished = true` if you want to include as matching candidates
+attractors that have vanished during the continuation process.
```

Comment:
Let's not duplicate this. Remove this whole copy pasted text in favor of just "matching works exactly as in [`RecurrencesFindAndMatch`] so see that docstring for more."